at-hoangtran/Week4/Final/Controllers/team_controller.rb

```diff
@@ -0,0 +1,31 @@
+require './Migrate/connect_db'
+require './Models/team_model'
+require 'pry'
+class TeamController
+  def self.add_team
+    print 'Team Name: '
+    team_name = gets.chomp
+    print 'Team Lead Id: '
+    team_lead_id = gets.chomp
+    print 'Total Member: '
+    total_member = gets.chomp
+    result = $client.query("INSERT INTO Team (name, employee_id, total_member) VALUES ('#{team_name}', '#{team_lead_id}', '#{total_member}')")
+  end
+
+  def self.add_member(team_id)
+    print 'ID Employee'
+    employee_id = gets.chomp
+  end
+
+  def self.choose_team
+    result = $client.query('SELECT * FROM Team')
+    @team = query.to_a
+  end
+
+  def self.show_team; end
+
+  def self.show_member
+    result = $client.query('SELECT * ')
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/8bc115c94428b11d6e3406f6ef06575baf3e6ffc/at-hoangtran/Week4/Final/Controllers/team_controller.rb#L28"></span>Lint/UselessAssignment: Useless assignment to variable - <code>result</code>. (https://github.com/rubocop-hq/ruby-style-guide#underscore-unused-vars)</td>
    </tr>
  </tbody>
</table>